internal/wrappers/access-management-http.go

```diff
@@ -0,0 +1,80 @@
+package wrappers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+)
+
+const (
+	// APIs
+	createAssignmentPath = "/"
+	entitiesForPath      = "/entities-for"
+	// EntityTypes
+	groupEntityType     = "group"
+	projectResourceType = "project"
+)
+
+type AccessManagementHTTPWrapper struct {
+	path          string
+	clientTimeout uint
+}
+
+func NewAccessManagementHTTPWrapper(path string) AccessManagementWrapper {
+	return &AccessManagementHTTPWrapper{
+		path:          path,
+		clientTimeout: viper.GetUint(commonParams.ClientTimeoutKey),
+	}
+}
+func (a *AccessManagementHTTPWrapper) CreateGroupsAssignment(projectID, projectName string, groups []*Group) error {
+	var resp *http.Response
+	for _, group := range groups {
+		assignment := CreateAssignment{
+			EntityID:     group.ID,
+			EntityType:   groupEntityType,
+			EntityName:   group.Name,
+			EntityRoles:  nil, // be used in the access-management phase 2
+			ResourceID:   projectID,
+			ResourceType: projectResourceType,
+			ResourceName: projectName,
+		}
+		params, err := json.Marshal(assignment)
+		if err != nil {
+			return errors.Wrapf(err, "Failed to parse request body")
+		}
+		path := fmt.Sprintf("%s/%s", a.path, createAssignmentPath)
+		resp, err = SendHTTPRequestWithJSONContentType(http.MethodPost, path, bytes.NewBuffer(params), true, a.clientTimeout)
+		if err != nil {
+			return errors.Wrapf(err, "Failed to create groups assignment")
```

Comment:
log? or debug log?

Reply:
I think it should be debug or even no print since we will have the error